### PR TITLE
[#507] Use job-id while querying for job lists

### PR DESCRIPTION
**Pull request by tonythomas01**
This would change select html to something that makes sense like: 

```
<select id="select" class="form-control" name="job_id">
	<option value="">-- Select Job --</option>
	<option value="3">another</option>
	<option value="5">another</option>
	<option value="4">test</option>
	<option value="1">test</option>
</select>
```

And things get handled at the backend correctly. Updated the tests as well. The dropdowns look as it were before

### CODE REVIEW

**Comment by tonythomas01** [comment]
Well - those failures seems unrelated. Possibly due to some issue with the travis runners ?

**Comment by tonythomas01** [comment]
Alright. PR Updated. Look at that thing in action - https://drive.google.com/open?id=0B8JDVmDxOfe5T0RkUll6Uk1JLWc 

Dynamic dropdowns with JQuery :sunglasses:

File: vms/administrator/views.py [hunk 1]
```diff
@@ -39,39 +39,53 @@ class ShowFormView(AdministratorLoginRequiredMixin, FormView):
...
+            **kwargs
+        )
+        context['event_list'] = get_events_ordered_by_name()
+        context['job_list'] = get_jobs_ordered_by_title()
+        context['organization_list'] = get_organizations_ordered_by_name()
+        return context
 
 
-class ShowReportListView(LoginRequiredMixin, AdministratorLoginRequiredMixin, ListView):
+class ShowReportListView(
```
[the remark this answers is not in the document]
Comment (tonythomas01) [reply vms/administrator/views.py]: wow. unused varaible `class` ? This bot should be killed with fire.

[the remark this answers is not in the document]
Comment (tapasweni-pathak) [reply vms/administrator/views.py]: I agree, plain noise. What is your plan for this pr @tonythomas01?

[the remark this answers is not in the document]
Comment (tonythomas01) [reply vms/administrator/views.py]: well, I will resolve the conflicts once I get time - and we should be able to get moving with this.

[the remark this answers is not in the document]
Comment (tapasweni-pathak) [reply vms/administrator/views.py]: dang @tonythomas01, will wait, ty for finding time to reply to this and dropping your comments in other pull requests.

**Comment by MeepyMay** [comment]
It's been over a month since this PR had activity. Closing for now.

**Pull request closed without merging**